pkg/k8s/namespaces/namespace.go

```diff
@@ -199,3 +229,111 @@ func (n *Namespaces) DestroySFSVolumes(ctx context.Context, ns string, opts Dele
 
 	return nil
 }
+
+func NewTrip(restConfig *rest.Config, opts *Options) (*Trip, error) {
```

Comment:
Minor thing, Can we include a comment in this function and `Wander` mentioning that were implemented to remove the dependency to the deleted module?